cmd/iop/root.go

```diff
@@ -37,7 +37,7 @@ type rootArgs struct {
 
 func addFlags(cmd *cobra.Command, rootArgs *rootArgs) {
 	cmd.PersistentFlags().StringVarP(&rootArgs.inFilename, "filename", "f", "",
-		"The path to the input IstioIstall CR. Uses in cluster value with kubectl if unset.")
+		"The path to the input Istio install CR. Uses in cluster value with kubectl if unset.")
```

Comment:
better to keep it as IstioInstall CR? seems like that's the CR name

Reply:
+1, you are right, will fix soon.

Reply:
Not quite sure we have a concrete CR named/typed as IstioInstall, but reverting it back to one word seems better. will do it today.